File: problem02-palindrome/palindrome.py
'''
@name palindrome 
@input str 
@desc The input string 
'''
def palindrome(str):
    reverseStr = str[::-1]
    if (reverseStr == str):
        return True 
    else:
        return False

print(palindrome('abc'))
print('\n')
print(palindrome('abcba'))

File: problem02-palindrome/palindrome.js
/**
 * @name palindrome
 * @desc For a given string, it checks whether the string is a plaindrome or not
 * @param str - The input string
 */
function palindrome(str) {
  const reversedStr = str.split('').reverse().join('');
  if (reversedStr === str) {
    return true;
  }
  return false;
}
